docs/reference/connector/apis/create-connector-api.asciidoc

```diff
@@ -66,19 +68,22 @@ Creates a connector document in the internal index and initializes its configura
 
 `index_name`::
 (Required, string) The target index for syncing data by the connector.
-
+// making this required does not align with the BYOI for connectors. I believe we discussed we're making this optional. 
+In Kibana, attaching an index also creates the index, if we specify an index here, will it be created? When? 
 `name`::
 (Optional, string) The name of the connector.
+// having an optional name Killed my Kibana experience. Without an index or a name, there is no way for me to go to connector Overview.
```

Comment:
Connector name comes with a set of validations in Kibana. What is the proposal for the next steps: relaxing Kibana validations or adding validations to the API?